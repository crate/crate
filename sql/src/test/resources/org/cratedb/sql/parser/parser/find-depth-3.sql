SELECT (1, (2,3))   IN ( 1 , 2 , 3 , (1, (2, 3)) )