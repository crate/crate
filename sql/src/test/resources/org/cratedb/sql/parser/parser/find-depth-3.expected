org.cratedb.sql.parser.parser.CursorNode@6b4bd192
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@3eabff46
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@e046ccf

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@44839139
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.InListOperatorNode@19b35853
			operator: IN
			methodName: in
			type: null
			leftOperand: 				
				org.cratedb.sql.parser.parser.RowConstructorNode@4d6bbe53
				depth: 2
			rightOperandList: 				
				org.cratedb.sql.parser.parser.RowConstructorNode@5c9c62da
				depth: 3
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@1f1e05b6